tkltest/generate/ui/ranked_attributes_short.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
Please move this file to the `test/ui` folder

Reply:
I have moved it to test/ui/helper_files